src/ports/postgres/modules/elastic_net/elastic_net.sql_in

```diff
@@ -130,7 +130,13 @@ the dependent variable expression to the <em>excluded</em> string.</DD>
 <DD>BOOLEAN, default: TRUE. Whether to normalize the data. Setting this to TRUE usually yields better results and faster convergence.</DD>
 
 <DT>grouping_col (optional)</DT>
-<DD>TEXT, default: NULL. <em>Not currently implemented. Any non-NULL value is ignored.</em> An expression list used to group the input dataset into discrete groups, running one regression per group. Similar to the SQL <tt>GROUP BY</tt> clause. When this value is NULL, no grouping is used and a single result model is generated.</DD>
+<DD>TEXT, default: NULL. 
+
+@note <em>Not currently implemented. Any non-NULL value is ignored.
+Grouping support will be added in a future release. </em>
+An expression list used to group the input dataset into discrete groups, 
```

Comment:
Maybe we shouldn't say anything about the expression list, since grouping isn't implemented yet.